### DISCUSSION TITLE
[BUG] Data randomly disappearing/lost from custom fields on the Product Model

### DISCUSSION THREAD

**Discussion by igoraugustynski** [swellstores/community#446] [Help]
Hi,
I have noticed another very worrying issue. We have some custom fields set up on the Product model.
<img width="1191" alt="Screenshot 2023-11-22 at 17 36 41" src="https://github.com/swellstores/community/assets/94485687/23cc2589-9fdd-46ce-96c0-040cff6f0cf3">

What is happening is that the data contained in those fields appears to get lost at random intervals. It stays there correctly for days, even months and suddenly disappears. This is a very important issue as it breaks our front end completely.

**Comment by logeshswell** [comment]
1. Value of which fields are being lost?
2. Can you share the product URL that is affected by this issue?
3. Do you remember the instance/action after which the data is lost?

**Reply by igoraugustynski** [reply]
1. The custom ones, in the screenshot I provided "Evento" is an example of this
2. It happens on all products randomly
3. I have no Idea

**Reply by igoraugustynski** [reply]
It just happened again so the issue is serious.

**Reply by igoraugustynski** [reply]
@logeshswell Please help us out on this, it is unacceptable.

**Reply by ericingram** [reply]
@igoraugustynski did you try looking at the /events records for those products to see when the data changed, and then try to correlate it with an API call?

Is the value still in the record but just not displaying in the UI?

Does it occur when saving a record?

**Reply by logeshswell** [reply]
Also, please share a sample product URL to investigate the root cause of this issue.
